authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,62 @@
+# Call Microsoft Graph from a Cloud Solution Provider application
+
+This topic describes how to enable application access to partner managed customer data via Microsoft Graph using either the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a> or the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service" target="_newtab">service to service client credentials flow</a>. 
+
+> **Note:** This topic applies **only** to Microsoft Cloud Solution Provider (CSP) application developers. The <a href="https://partner.microsoft.com/en-US/cloud-solution-provider" target="_newtab">Microsoft Cloud Solution Provider (CSP)</a> program enables Microsoft’s partners to resell and manage Microsoft Online services to customers. 
+
+**Important**: Calling Microsoft Graph from a CSP application is only supported for directory resources (such as **user**, **group**,**device**, **organization**) and [Intune](..\beta\resources\intune_graph_overview) resources.
+
+## What is a Partner Managed Application?
+The CSP program enables Microsoft’s partners to resell and manage Microsoft Online services such as Office 365, Azure and CRM Online) to customers. Management of customer services is done through Delegated Admin Privileges (DAP) which enables designated partner users and agents to access and configure their customers’ environments.
+
```

Comment:
Acronyms can sometimes be trademarks of other companies (http://www.dap.com/). I couldn't find the DAP acronym used across MS sites. If unsure, we can check with CELA. More info here: https://microsoft.sharepoint.com/sites/lcaweb/Home/Copyrights-Trademarks-and-Patents/Trademarks/Acronyms-Abbreviations#